pkg/volume/csi/csiv0/csi.pb.go

```diff
@@ -19,13 +19,17 @@ limitations under the License.
 
 package csiv0
 
-import proto "github.com/golang/protobuf/proto"
-import fmt "fmt"
-import math "math"
-import wrappers "github.com/golang/protobuf/ptypes/wrappers"
-
 import (
+	fmt "fmt"
```

Comment:
still seeing a diff here... can you actually drop this file from the commit?

Reply:
sorry, OK I have done it Is Any Problems Now ?

Reply:
@liggitt  @mattjmcnaughton